elastic/kibana
# [Synthetics] Filters tls certs by saved objects

## Summary

### Testing

1. Create few http monitors in default space
2. Make sure you can see tls certificates related to those in tls certs page
3. Switch to some other space, and you shouldn't see tls certs related to default space in that other space

<img width="1728" alt="image" src="https://github.com/elastic/kibana/assets/3505601/00979217-334b-4f94-b07a-9f47ec249bcd">
